src/App.tsx

```diff
@@ -1,173 +1,89 @@
-import { Loader } from './components/Loader';
+import {
+  Navigate, Route, Routes,
+} from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { PeopleTable } from './components/PeopleTable';
+import { PageNav } from './components/PageNav';
+import { ErrorNotification } from './components/ErrorNotification';
 
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+import { Person } from './types/Person';
+import { getPeople } from './api';
+
+export const App = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoaded, setIsPeopleLoaded] = useState(false);
+  const [isErrorShown, setIsErrorShown] = useState(false);
```

Comment:
and here the name hints at the fact that once the error was shown once or not.  this [link](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) can be useful for your renaming
```suggestion
  const [isShowError, setIsShowError] = useState(false);
```